internal/cortex/chunk/encoding/varbit_helpers.go

```diff
@@ -16,7 +16,7 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-//nolint //Since this was copied from Prometheus leave it as is
+// nolint //Since this was copied from Prometheus leave it as is
```

Comment:
There are a bunch of whitespace changes - are they needed?

Reply:
those are from `go fmt ./...`.